### ISSUE TITLE
CodeLens does not appear on conversion operators

### ISSUE THREAD

**Issue by jhinder** [dotnet/roslyn#30210]
**Version Used**: VS 15.8.5 (Roslyn 2.9.0.63208)

**Steps to Reproduce**:

1. Create a type with a conversion operator.

**Expected Behavior**: CodeLens appears for the operator, like with all other operators.

**Actual Behavior**: CodeLens does not appear for the operator.

![struct-operator](https://user-images.githubusercontent.com/823398/46204954-4ee78b80-c31f-11e8-9cab-53b01b156e36.PNG)

**Comment by jinujoseph** [comment]
@heejaechang  this expected ?

**Comment by heejaechang** [comment]
I don't know. but since we don't provide "Find All References" on those operators (explicit and implicit), we probably don't provide on for code lens?

@mavasani worked on this FAR recently, do you have an idea? since code lens sit on top of FAR, if FAR doesn't support it, then code lens probably won't as well.

**Comment by heejaechang** [comment]
FAR not supporting those implict or explicit operator, not sure whether this is by design or just a bug.
tagging @dotnet/roslyn-ide 

anyway, looks like right now, it is not supported.

**Comment by mavasani** [comment]
@CyrusNajmabadi might now as well. I believe the finder is [here](http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/FindSymbols/FindReferences/Finders/OperatorSymbolReferenceFinder.cs,12)

**Comment by CyrusNajmabadi** [comment]
That only finds normal operators.  We don't have any way to index or find things like implicit conversions (since our indices are syntactic).

**Comment by heejaechang** [comment]
yep. the normal operator works fine. it is implicit/explicit operator that don't work right now. so, I guess this is by design for now. and we could open an issue as feature request. @jinujoseph ?
